open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -25,7 +25,7 @@
     "import matplotlib.pyplot as plt\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.            assert test_df.iloc[1, 1] == 9
</pre><p>You should compare the entire pandas series, instead of a single value. Please apply this to all test cases.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>DONE</p>